extensions/3DTILES_layers/0.0.0/schema/3DTILES_layers.implicit_tiling_content_availability.schema.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Implicit tiling layer availability object.",
+  "type": "object",
+  "properties": {
+    "bufferView": {
```

Comment:
@lilleyse is the intent here that this extension depends on `3DTILES_layers` which in turn depends on `3DTILES_binary_buffers`? or is it that this extension depends on both?

Reply:
Layer content availability, like tile availability and content availability, refers to the buffers/bufferViews that are in the subtree JSON, so no extension needed. 3DTILES_binary_buffers is just for tileset.json while this new subtree JSON is its own self contained thing.